corehq/apps/app_manager/suite_xml/features/case_tiles.py

```diff
@@ -88,11 +87,14 @@ def build_case_tile_detail(self):
 
             for column_info in self.detail_column_infos:
                 # column_info is an instance of DetailColumnInfo named tuple.
-                style = Style(grid_x=column_info.column.grid_x, grid_y=column_info.column.grid_y,
-                              grid_height=column_info.column.height, grid_width=column_info.column.width,
-                              horz_align=column_info.column.horizontal_align,
-                              vert_align=column_info.column.vertical_align,
-                              font_size=column_info.column.font_size)
+                style = None
+                if column_info.column.grid_x or column_info.column.grid_y \
+                        or column_info.column.height or column_info.column.width:
```

Comment:
actually it just occurred to me that these are integer fields, so they could be set to `0` which would evaluate as False.  You might want to do something like:
```python
if any(field is not None for field in [...]):
```

Reply:
changed